app/views/devise/two_factor_authentication/show_totp.html.slim

```diff
@@ -2,10 +2,11 @@
 
 h1.heading = t('devise.two_factor_authentication.totp_header_text')
 p
-  'Please enter the code from your authenticator app.
-  'If you have several accounts set up in your app, enter the code
-  'corresponding to <strong>#{current_user.email}</strong> at
-  '<strong>Login.gov</strong>.
+  | Please enter the code from your authenticator app.
+    If you have several accounts set up in your app, enter the code
+    corresponding to <strong>#{current_user.email}</strong> at
+    <strong>Login.gov</strong>.
+
```

Comment:
This PR doesn't seem to be dependent on these style changes. It's not a big deal to me, but while you're making changes here, do you also want to updated `Login.gov` to `#{APP_NAME}`?